Plugins/GeoModel/CMakeLists.txt

```diff
@@ -22,7 +24,7 @@ target_include_directories(
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
 target_link_libraries(
     ActsPluginGeoModel
-  PUBLIC ActsCore GeoModelKernel GeoModelIO::GeoModelDBManager GeoModelIO::GeoModelRead)
+  PUBLIC ActsCore GeoModelCore::GeoModelKernel GeoModelCore::GeoModelHelpers GeoModelIO::GeoModelDBManager GeoModelIO::GeoModelRead)
```

Comment:
[CI](https://github.com/acts-project/acts/actions/runs/10160619684/job/28097351896?pr=3268) is complaining about `GeoModelHelpers` not existing @Matthewharri - is that necessary? does that exist in v6.3.0?

Reply:
Yeah this was not necessary. I left it in by accident, should be all better now.